weasyl/controllers/user.py

```diff
@@ -99,12 +99,13 @@ def GET(self):
     def POST(self):
         form = web.input(
             username="", password="", passcheck="", email="", emailcheck="",
-            day="", month="", year="", recaptcha_challenge_field="", recaptcha_response_field="")
+            day="", month="", year="", recaptcha_challenge_field="", 
+            g_recaptcha_response=web.input("g-recaptcha-response"))
 
         if not define.captcha_verify(form):
             return define.errorpage(
                 self.user_id,
-                "The image verification you entered was not correct; you should go back and try again.")
+                "There was an error verifying the data you entered; you should go back and try again.")
```

Comment:
Okay, so in trying this out, I realized this error message is way too vague. It should reference the captcha.